src/main/java/com/continuum/nova/chunks/ChunkBuilder.java

```diff
@@ -74,17 +68,17 @@ public void createMeshesForChunk(ChunkUpdateListener.BlockUpdateRange range) {
      * Adds the block at the given position to the blocksForFilter map under each filter that matches the block
      *
      * @param blocksForFilter The map of blocks to potentially add the given block to
-     * @param pos The position of the block to add
+     * @param pos             The position of the block to add
      */
     private void filterBlockAtPos(Map<String, List<BlockPos>> blocksForFilter, BlockPos pos) {
         IBlockState blockState = world.getBlockState(pos);
-        if(blockState.getRenderType().equals(EnumBlockRenderType.INVISIBLE)) {
+        if (blockState.getRenderType() == EnumBlockRenderType.INVISIBLE) {
```

Comment:
Why?

Reply:
Because enums can be compared with "==" and it is null safe.

Reply:
Looks like == is used everywhere else, so I'll accept this change